website/pages/style.css

```diff
@@ -22,26 +23,39 @@
 @import '~@hashicorp/react-product-downloader/dist/style.css';
 @import '~@hashicorp/react-vertical-text-block-list/dist/style.css';
 @import '~@hashicorp/react-docs-sidenav/dist/style.css';
-@import '~@hashicorp/react-case-study-slider/dist/style.css';
 @import '~@hashicorp/react-content/dist/style.css';
+@import '~@hashicorp/react-subnav/dist/style.css';
 @import '~@hashicorp/react-text-and-content/dist/style.css';
 @import '~@hashicorp/react-code-block/dist/style.css';
 @import '~@hashicorp/react-consent-manager/dist/style.css';
 @import '~@hashicorp/react-alert/dist/style.css';
+@import '~@hashicorp/react-call-to-action/dist/style.css';
+@import '~@hashicorp/react-text-split/dist/style.css';
+@import '~@hashicorp/react-text-split-with-code/dist/style.css';
 
 /* Global Components - Transpiled */
 @import '~@hashicorp/react-mega-nav/style.css';
```

Comment:
This can be dropped if sticking with the local copy of `mega-nav`

Reply:
Good catch here; we'll drop this for initial launch, as we're going to keep the ejected component for now.